### ISSUE TITLE
Pugna tp'ing bug

### ISSUE THREAD

**Issue by Syfroenix** [ValveSoftware/Dota2-Gameplay#17955]
### Ability name

Pugna innate ability cancels teleport

### Description

Pugna can now cast while channeling. However, when the "stop or hold required for teleport cancel" is selected, this innate ability overrides this and tries to cast a spell, cancelling the teleport

### Example Match ID (and possibly Timestamp)

7751534609 Time: 38:35

**Comment by flare-ws** [comment]
possible duplicate of https://github.com/ValveSoftware/Dota2-Gameplay/issues/17879

**Comment by Syfroenix** [comment]
@flare-ws This is a different bug to that one, thanks

**Comment by flare-ws** [comment]
btw, as per the description of the innate, pugna's spells shouldnt cancel the tp, regardless of the option.

**Comment by qudesn1q** [comment]
Yes. The option to cancel teleport using a separate key does not work on Pugna. And this still hasn't been fixed.

**Comment by flare-ws** [comment]
What pisses me off the most is the fact that while you play pugna you are unable to shift+dagger while you are casting tp scroll

**Comment by Zdotafeedback** [comment]
Played my first Pugna game since this facet was added the other day and I screwed up TP engagements 4 times due to this bug. It's interesting to realise just how much I relied on queueing abilities directly after TP.

I think it's very important this gets fixed.
